packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationMembers.tsx

```diff
@@ -72,16 +76,16 @@ export const OrganizationMembers = withCardStateProvider(() => {
                   width: '100%',
                 }}
               >
-                {isAdmin && __unstable_manageBillingUrl && <MembershipWidget />}
+                {canManageMemberships && __unstable_manageBillingUrl && <MembershipWidget />}
                 <ActiveMembersList />
               </Flex>
             </TabPanel>
-            {isAdmin && (
+            {canManageMemberships && (
```

Comment:
Is it possible to use the <Gate /> component here instead?

Reply:
Sadly no, because TabPanel needs to be direct child of TabPanels